packages/libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -659,15 +660,17 @@ export class ChatsApi
    * @returns the rpc object
    */
   public async permit(params: ChatPermitRequest) {
-    const { currentUserId, permit } = await parseParams(
+    const { currentUserId, permit, permitList, allow } = await parseParams(
       'permit',
       ChatPermitRequestSchema
     )(params)
     return await this.sendRpc({
       current_user_id: currentUserId,
       method: 'chat.permit',
       params: {
-        permit
+        permit: permit ?? ChatPermission.ALL,
+        permit_list: permitList ?? [ChatPermission.ALL],
+        allow
```

Comment:
what does allow do?

Reply:
allow is the flag that tells the backend we're using the new permit_list model now